IPs/RSKIP346.md

```diff
@@ -0,0 +1,60 @@
+---
+rskip: 346
```

Comment:
Please replace all the RSKIP327 references with this one. For example, the description of the PR and the first commit are referencing the old RSKIP number.